rfcs/20251103-level-zero-runtime/README.md

```diff
@@ -0,0 +1,184 @@
+# Support for Level Zero GPU runtime (RFC)
+
+## Introduction
+
+This RFC discusses the option of adding the Intel® oneAPI Level Zero (Level Zero) as fully supported oneDNN GPU runtime.
+
+Level Zero is a fully featured API that provides direct-to-metal interfaces to offload Intel® accelerator devices (similarly to OpenCL™). As such, it is the preferred API for providing explicit device controls for runtime APIs and libraries using Intel® GPUs. For example, Level Zero is the default backend of SYCL runtime.
+
+For oneDNN, Level Zero would provide quicker access to the newest features available for Intel® accelerators, including: reusable and mutable command lists, support for multi-device contexts, shared memory between accelerators, as well as lowered execution latency.
+
+Using a modified [oneDNN example](https://github.com/uxlfoundation/oneDNN/blob/main/examples/cnn_inference_f32.cpp) and [POC implementation](https://github.com/uxlfoundation/oneDNN/tree/spalicki/l0_backend), the Level Zero runtime shows better performance than OpenCL™ and SYCL:
+
+|           | Level Zero | OpenCL™ | SYCL with Level Zero | SYCL with OpenCL™ |
+| --------- | ---------- | ------- | -------------------- | ----------------- |
+| Min (ms): |        557 |     640 |                  599 |               658 |
+| Avg (ms): |        579 |     668 |                  621 |               689 |
+
+## Proposal
+
+### Build option
+
+Adding Level Zero runtime does not require new build options and only requires a new runtime `L0` in the existing CMake build option `ONEDNN_GPU_RUNTIME`, and it and assumes that: `ONEDNN_GPU_VENDOR=INTEL`.
+
+### API/ABI change
+
+Adding Level Zero as a new oneDNN runtime is fairly invisible to the end user and requires no API/ABI modification.
```

Comment:
I guess this contradict the part about interop API, where new APIs are needed for interop purposes. Maybe merge this section with interop section?